package.json

```diff
@@ -130,6 +131,7 @@
     "jsonschema": "^1.2.4",
     "lodash": "^4.17.19",
     "loglevel": "^1.4.1",
+    "loose-envify": "^1.4.0",
```

Comment:
This should be under `devDependencies`, since it's used by the build system. Also, shouldn't this replace `envify`? I noticed that `envify` is still listed